server/repositories/tests.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     let queries = [];
 
     tests.forEach(test => {
+      // FIXME
```

Comment:
One way to fix 1. could be to lookup which test-id's the page already have, and then delete those that are not in the tests-array.